filesystem/local_copy.cpp

```diff
@@ -0,0 +1,80 @@
+#include "local_copy.h"
+
+#include "../common/log.h"
+#include "../proto/messages.pb.h"
+#include "tcp.h"
+
+#include <map>
+#include <string>
+
+std::map<std::string, std::string> copies_store;
+
+void ensure_local_copy_initialized(const int socket, const char *path) {
```

Comment:
Shouldn't it return some kind of error since there are error logs in this function?